pkg/butler/configure/setup_chassis.go

```diff
@@ -217,7 +217,7 @@ func (b *CmcSetup) Post() {
 // and powers it back on.
 func (b *CmcSetup) ensurePoweredUp() (err error) {
 	status, _ := b.chassis.IsOn()
-	if status == false {
+	if !status {
```

Comment:
As a follow-up this should probably be changed to "status_on", "powerOn", or similar so the test reads more clearly.